sql/moz-fx-data-shared-prod/firefox_accounts/fxa_content_auth_stdout_events/view.sql

```diff
@@ -122,5 +122,6 @@ SELECT
   REPLACE(JSON_EXTRACT(event_properties, '$.email_version'), "\"", "") AS email_version,
   REPLACE(JSON_EXTRACT(event_properties, '$.plan_id'), "\"", "") AS plan_id,
   REPLACE(JSON_EXTRACT(event_properties, '$.product_id'), "\"", "") AS product_id,
+  REPLACE(JSON_EXTRACT(event_properties, '$.promotionCode'), """, "") AS promotion_code,
```

Comment:
missing an escape `\`
```suggestion
  REPLACE(JSON_EXTRACT(event_properties, '$.promotionCode'), "\"", "") AS promotion_code,
```